duckduckgo/iOS
# Add notes to autofill logins

Task/Issue URL: https://app.asana.com/0/0/1203021186566017/f, https://app.asana.com/0/0/1202809065048065/f
Tech Design URL:
CC:

**Description**:
Linked to this BSK PR: https://github.com/duckduckgo/BrowserServicesKit/pull/142
Updates BSK to include notes support, and changes the login management interface to have a notes field.



**Steps to test this PR**:
1. Edit an existing login to have notes
2. Create a new login with a note. 
3. Create a new login with a very long note, and check that the notes field eventually stops growing. The view should truncate, or if in edit mode should scroll.


**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
